zk_toolbox/crates/zk_inception/src/commands/ecosystem/args/create.rs

```diff
@@ -40,7 +40,7 @@ impl EcosystemCreateArgs {
 
         let link_to_code = self.link_to_code.unwrap_or_else(|| {
             let link_to_code_selection = PromptSelect::new(
-                "Select the origin of zksync-era repository",
+                "Add the (recommended) to the clone option",
```

Comment:
Badly explained this one. This sentence looks good, but add a "(recommended)" to the end of the clone option